sdk/search/Azure.Search.Documents/tests/Samples/Readme.cs

```diff
@@ -49,6 +54,25 @@ public async Task Authenticate()
             #endregion Snippet:Azure_Search_Tests_Samples_Readme_Authenticate
         }
 
+        [Test]
+        [SyncOnly]
+        public async Task AuthenticateWithAAD()
+        {
+            await using SearchResources resources = await SearchResources.GetSharedHotelsIndexAsync(this, true);
+            Environment.SetEnvironmentVariable("SEARCH_ENDPOINT", resources.Endpoint.ToString());
```

Comment:
Tests run in parallel (from different assemblies) - maybe not important now, but you should avoid setting env vars that other code uses. Instead, consider using your actual search endpoint for the real test (probably from your `TestEnvironment`), and use `SNIPPET` symbol only for snippets, e.g.,

```c#
#if SNIPPET
Uri endpoint = new(Environment.GetEnvironmentVariable("SEARCH_ENDPOINT"));
#else
Uri endpoint = new(TestEnvironment.SearchEndpoint);
#endif
```